src/modules/auth/components/authenticated-route.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Route, Redirect } from 'react-router-dom';
+
+import { AUTHENTICATION } from 'modules/app/constants/views';
+
+import makePath from 'modules/app/helpers/make-path';
+
+import getValue from 'utils/get-value';
+
+const AuthenticatedRoute = ({ component: Component, isLogged, ...rest }) => (
```

Comment:
That’s basically there for convention consistency.  

A little further down you’ll see the use of `Component`, if this wasn’t there, we’d have to do something to the affect of `<props.component…` instead of `<Component…`.